syntax = "proto3";

message PrintBalanceRsp {
  int32 clusterId = 1; // Cluster that server belong to
  int32 serverId = 2; // Server's id
  int32 dataItemID = 3; // Data item that queried
  int32 balance = 4; // Balance of data item
}